mobile/lib/widgets/settings/language_settings.dart

```diff
@@ -245,13 +248,13 @@ class _LanguageApplyButton extends StatelessWidget {
                       strokeWidth: 2,
                     ),
                   )
-                : Text(
-                    'setting_languages_apply'.tr(),
-                    style: const TextStyle(
+                : const Text(
+                    'setting_languages_apply',
```

Comment:
```suggestion
                    'setting_languages_apply'.t(context: context),
```

I believe this would make the function call similar to others in this file